### DISCUSSION TITLE
output.css has 3 warnings

### DISCUSSION THREAD

**Discussion by JalalHamed** [tailwindlabs/tailwindcss#14136] [Help]
Hi 👋,

I bootstrapped tailwind using its stand-alone CLI (windows OS), created an `input.css` file and added the tailwind directives to it and ran `./tailwindcss -i input.css -o output.css --watch --minify` command in terminal (using either of the flags alone doesn't make a difference). And the output.css that is being generated has 3 warnings in it:

![Screenshot](https://github.com/user-attachments/assets/d925f447-53fe-405e-85df-89e7f40d695e)

Two of which are a compatibility concern (using a nonstandard property according to the tooltip):

![Screenshot](https://github.com/user-attachments/assets/f58a99e3-ec36-4d26-b929-811a34710d11)

And the last one is a property being ignored due to the wrong display given:

![Screenshot](https://github.com/user-attachments/assets/b51f4e7d-7761-4a22-9621-b07feb0f3174)

Couldn't find a similar issue through my search so I was wondering am I doing something wrong? Or is it a legit issue here?

**Comment by wongjn** [comment]
I believe you can ignore these issues.

**Reply by JalalHamed** [reply]
Sure I can but wouldn't fixing them be a better solution?

**Reply by wongjn** [reply]
I believe the declarations are like that by design. [See the annotated source file](https://github.com/tailwindlabs/tailwindcss/blob/v3.4.7/src/css/preflight.css) for reasoning for each declaration.

**Comment by JalalHamed** [comment]
found a related comment:
https://github.com/jensimmons/cssremedy/issues/14#issuecomment-634849770
